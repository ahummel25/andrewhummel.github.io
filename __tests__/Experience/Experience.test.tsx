import React from 'react';
import { create } from 'react-test-renderer';
import { GatsbyImage } from 'gatsby-plugin-image';

import Experience from '../../src/components/Experience';

describe('Experience', () => {
  it('renders correctly', () => {
    const tree = create(<Experience />);

    expect(tree).toBeDefined();
  });

  it('renders the correct number of images', () => {
    const tree = create(<Experience />);

    const images = tree.root.findAllByType(GatsbyImage);

    expect(images.length).toBe(4);
  });
});
