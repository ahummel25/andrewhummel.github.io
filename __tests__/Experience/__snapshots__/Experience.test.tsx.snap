// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience renders correctly 1`] = `
.emotion-0 {
  font-family: Roboto,sans-serif;
  margin: 0 auto;
  background-color: rgb(0, 128, 115);
  font-weight: 100;
  padding: 30px 40px;
  margin-bottom: 60px;
  text-transform: uppercase;
}

@media (max-width: 992px) {
  .emotion-0 {
    padding: 20px 30px;
    margin-bottom: 40px;
  }
}

.emotion-17 {
  width: 95%;
  font-family: Roboto,sans-serif;
  font-weight: 300;
  padding: 0 24px;
  max-width: 1280px;
  letter-spacing: 0.25px;
}

@media (min-width: 1200px) {
  .emotion-17 {
    width: 860px;
  }
}

.emotion-4 {
  margin-bottom: 60px;
}

.emotion-4 .card-footer {
  background-color: rgba(0, 0, 0, 0.02);
  border: none;
  padding: 20px;
}

.emotion-4 .card-footer span {
  color: rgb(102, 102, 102);
  font-size: 94%;
  font-weight: 400;
}



.emotion-3 .card-title {
  padding-top: 10px;
  text-transform: uppercase;
}

@media (max-width: 768px) {
  .emotion-3 .card-title {
    text-align: center;
  }
}

.emotion-3 .divider {
  margin: 0;
}

.emotion-3 .hoverline {
  border-bottom: 3px solid transparent;
  padding: 3px 3px 0 3px;
  -webkit-transition: 0.25s all;
  transition: 0.25s all;
  -moz-transition: 0.25s all;
  -webkit-transition: 0.25s all;
}

.emotion-3 .hoverline:hover {
  border-bottom: 3px solid rgb(0, 150, 136);
}

.emotion-3 .text-description {
  color: rgb(102, 102, 102);
}

.emotion-3 .subtitle {
  font-size: 16px;
  font-weight: 400;
  padding-top: 20px;
}

.emotion-3 h6 {
  font-size: 1.05rem;
  font-weight: 300;
  letter-spacing: 1px;
  line-height: 1.1rem;
  margin-top: 2rem;
  text-transform: uppercase;
}

.emotion-3 ul>li {
  line-height: 150%;
  font-size: 14px;
  font-weight: 400;
  margin-left: 20px;
  list-style-type: disc;
}

.emotion-2 {
  margin-bottom: 0;
  padding-bottom: 10px;
}

.emotion-2 .role {
  font-size: 1.25rem;
  font-weight: 400;
  position: absolute;
  right: 38px;
  top: 35px;
}

@media (max-width: 768px) {
  .emotion-2 .role {
    position: relative;
    right: initial;
    top: initial;
    text-align: center;
  }
}

.emotion-1 {
  max-height: 100px;
  height: 60px;
}

@media (max-width: 768px) {
  .emotion-1 {
    margin-bottom: 10px;
  }
}

.emotion-0 {
  font-family: Roboto,sans-serif;
  margin: 0 auto;
  background-color: rgb(0, 128, 115);
  font-weight: 100;
  padding: 30px 40px;
  margin-bottom: 60px;
  text-transform: uppercase;
}

@media (max-width: 992px) {
  .emotion-0 {
    padding: 20px 30px;
    margin-bottom: 40px;
  }
}

.emotion-17 {
  width: 95%;
  font-family: Roboto,sans-serif;
  font-weight: 300;
  padding: 0 24px;
  max-width: 1280px;
  letter-spacing: 0.25px;
}

@media (min-width: 1200px) {
  .emotion-17 {
    width: 860px;
  }
}

.emotion-4 {
  margin-bottom: 60px;
}

.emotion-4 .card-footer {
  background-color: rgba(0, 0, 0, 0.02);
  border: none;
  padding: 20px;
}

.emotion-4 .card-footer span {
  color: rgb(102, 102, 102);
  font-size: 94%;
  font-weight: 400;
}



.emotion-3 .card-title {
  padding-top: 10px;
  text-transform: uppercase;
}

@media (max-width: 768px) {
  .emotion-3 .card-title {
    text-align: center;
  }
}

.emotion-3 .divider {
  margin: 0;
}

.emotion-3 .hoverline {
  border-bottom: 3px solid transparent;
  padding: 3px 3px 0 3px;
  -webkit-transition: 0.25s all;
  transition: 0.25s all;
  -moz-transition: 0.25s all;
  -webkit-transition: 0.25s all;
}

.emotion-3 .hoverline:hover {
  border-bottom: 3px solid rgb(0, 150, 136);
}

.emotion-3 .text-description {
  color: rgb(102, 102, 102);
}

.emotion-3 .subtitle {
  font-size: 16px;
  font-weight: 400;
  padding-top: 20px;
}

.emotion-3 h6 {
  font-size: 1.05rem;
  font-weight: 300;
  letter-spacing: 1px;
  line-height: 1.1rem;
  margin-top: 2rem;
  text-transform: uppercase;
}

.emotion-3 ul>li {
  line-height: 150%;
  font-size: 14px;
  font-weight: 400;
  margin-left: 20px;
  list-style-type: disc;
}

.emotion-2 {
  margin-bottom: 0;
  padding-bottom: 10px;
}

.emotion-2 .role {
  font-size: 1.25rem;
  font-weight: 400;
  position: absolute;
  right: 38px;
  top: 35px;
}

@media (max-width: 768px) {
  .emotion-2 .role {
    position: relative;
    right: initial;
    top: initial;
    text-align: center;
  }
}

.emotion-1 {
  max-height: 100px;
  height: 60px;
}

@media (max-width: 768px) {
  .emotion-1 {
    margin-bottom: 10px;
  }
}

<section
  className="section scrollspy"
  id="experience"
>
  <h3
    className="page-title white-text emotion-0"
  >
    Experience
  </h3>
  <div
    className="container emotion-17"
    width={860}
  >
    <div
      className="card emotion-4"
    >
      <div
        className="card-content emotion-3"
      >
        <div
          className="row emotion-2"
        >
          <div
            className="col s12 m2 emotion-1"
          >
            <a
              href="https://inspire11.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "maxHeight": "100%",
                    "maxWidth": null,
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  aria-hidden={true}
                  style={
                    Object {
                      "paddingBottom": "63.83896477354421%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEj"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "contain",
                      "objectPosition": "center",
                      "opacity": 1,
                      "position": "absolute",
                      "top": 0,
                      "transitionDelay": "500ms",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 800px) 100vw, 800px"
                    srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/i11.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/i11.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/i11.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/i11.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/i11.jpg 1391w"
                  />
                  <img
                    alt="Inspire11 Logo"
                    loading="lazy"
                    onLoad={[Function]}
                    sizes="(max-width: 800px) 100vw, 800px"
                    src="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/i11.jpg"
                    srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/i11.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/i11.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/i11.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/i11.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/i11.jpg 1391w"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "contain",
                        "objectPosition": "center",
                        "opacity": 0,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 500ms",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/i11.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/i11.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/i11.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/i11.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/i11.jpg 1391w\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/i11.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/i11.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/i11.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/i11.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/i11.jpg 1391w\\" src=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/i11.jpg\\" alt=\\"Inspire11 Logo\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </div>
          <div
            className="card-title"
          >
            <a
              className="teal-text hoverline"
              href="https://inspire11.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              Inspire11
            </a>
          </div>
          <div
            className="role brown-text"
          >
            Full Stack Software Developer
          </div>
        </div>
        <hr
          className="MuiDivider-root divider MuiDivider-middle"
        />
        <p
          className="subtitle"
        >
          <em
            className="text-description"
          >
            Innovative, software consulting firm implementing cutting-edge solutions combining technology, strategy, and change management to meet and exceed client needs
          </em>
        </p>
        <h6>
          Client Product Development
        </h6>
        <ul>
          <li>
            Contribute core features to a client requested video streaming progressive web application (PWA)
          </li>
          <li>
            Collaborated with agile team to build front-end of PWA using React
          </li>
          <li>
            Researched and determined best third party video streaming software to fit client needs
          </li>
          <li>
            Lead developer on building out AWS lambda functions to handle streaming using AWS SDK
          </li>
          <li>
            Tools: React, Typescript, SASS // Node.js, Typescript, Go, MySQL // AWS
          </li>
        </ul>
      </div>
      <div
        className="card-footer"
      >
        <span>
          May 2019 - Present | Chicago, IL
        </span>
      </div>
    </div>
    <div
      className="card emotion-4"
    >
      <div
        className="card-content emotion-3"
      >
        <div
          className="row emotion-2"
        >
          <div
            className="col s12 m2 emotion-1"
          >
            <a
              href="https://juristat.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "maxHeight": "100%",
                    "maxWidth": null,
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  aria-hidden={true}
                  style={
                    Object {
                      "paddingBottom": "63.83896477354421%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEj"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "contain",
                      "objectPosition": "center",
                      "opacity": 1,
                      "position": "absolute",
                      "top": 0,
                      "transitionDelay": "500ms",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 800px) 100vw, 800px"
                    srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/juristat.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/juristat.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/juristat.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/juristat.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/juristat.jpg 1391w"
                  />
                  <img
                    alt="Juristat Logo"
                    loading="lazy"
                    onLoad={[Function]}
                    sizes="(max-width: 800px) 100vw, 800px"
                    src="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/juristat.jpg"
                    srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/juristat.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/juristat.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/juristat.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/juristat.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/juristat.jpg 1391w"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "contain",
                        "objectPosition": "center",
                        "opacity": 0,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 500ms",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/juristat.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/juristat.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/juristat.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/juristat.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/juristat.jpg 1391w\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/juristat.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/juristat.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/juristat.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/juristat.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/juristat.jpg 1391w\\" src=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/juristat.jpg\\" alt=\\"Juristat Logo\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </div>
          <div
            className="card-title"
          >
            <a
              className="teal-text hoverline"
              href="https://www.juristat.com/"
              rel="noopener noreferrer"
              target="_blank"
            >
              Juristat
            </a>
          </div>
          <div
            className="role brown-text"
          >
            Software Engineer
          </div>
        </div>
        <hr
          className="MuiDivider-root divider MuiDivider-middle"
        />
        <p
          className="subtitle"
        >
          <em
            className="text-description"
          >
            Legal software company allowing patent prosecutors to predict the behavior of their examiner, competitors, and potential clients using big data analytics
          </em>
        </p>
        <h6>
          Core API Development
        </h6>
        <ul>
          <li>
            Built and enhanced Node.js web APIs by designing and writing Javascript functions and complex GraphQL schema types
          </li>
          <li>
            Creating and maintaining data in multiple Elasticsearch indexes via HTTP requests and Node.js APIs.
          </li>
          <li>
            Leveraging Node’s AWS SDK to perform operations in S3, SQS, and other services.
          </li>
          <li>
            Writing extensive unit tests using Jest and Mocha and performing QA prior to production releases
          </li>
          <li>
            Tools: Node.js, Typescript, GraphQL // Elasticsearch, MySQL // AWS
          </li>
        </ul>
      </div>
      <div
        className="card-footer"
      >
        <span>
          July 2018 - May 2019 | St. Louis, MO
        </span>
      </div>
    </div>
    <div
      className="card emotion-4"
    >
      <div
        className="card-content emotion-3"
      >
        <div
          className="row emotion-2"
        >
          <div
            className="col s12 m2 emotion-1"
          >
            <a
              href="https://www.wwt.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "maxHeight": "100%",
                    "maxWidth": null,
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  aria-hidden={true}
                  style={
                    Object {
                      "paddingBottom": "63.83896477354421%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEj"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "contain",
                      "objectPosition": "center",
                      "opacity": 1,
                      "position": "absolute",
                      "top": 0,
                      "transitionDelay": "500ms",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 800px) 100vw, 800px"
                    srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/wwt.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/wwt.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/wwt.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/wwt.jpg 1391w"
                  />
                  <img
                    alt="WWT Logo"
                    loading="lazy"
                    onLoad={[Function]}
                    sizes="(max-width: 800px) 100vw, 800px"
                    src="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg"
                    srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/wwt.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/wwt.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/wwt.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/wwt.jpg 1391w"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "contain",
                        "objectPosition": "center",
                        "opacity": 0,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 500ms",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/wwt.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/wwt.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/wwt.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/wwt.jpg 1391w\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/wwt.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/wwt.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/wwt.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/wwt.jpg 1391w\\" src=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg\\" alt=\\"WWT Logo\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </div>
          <div
            className="card-title"
          >
            <a
              className="teal-text hoverline"
              href="https://www.wwt.com/"
              rel="noopener noreferrer"
              target="_blank"
            >
              World Wide Technology
            </a>
          </div>
          <div
            className="role brown-text"
          >
            Senior Developer
          </div>
        </div>
        <hr
          className="MuiDivider-root divider MuiDivider-middle"
        />
        <p
          className="subtitle"
        >
          <em
            className="text-description"
          >
            Technology and supply chain services provider
          </em>
        </p>
        <h6>
          Software Engineering
        </h6>
        <ul>
          <li>
            Supporting and customizing Oracle's Warehouse Management system (Inventory (INV) and Shipping (WSH) modules) to fit user needs and requirements
          </li>
          <li>
            Writing custom Java code on Oracle's Mobile Web Applications (MWA) framework to enhance WMS system.
          </li>
          <li>
            Enhancing existing web applications written in AngularJS, Groovy/Grails, and PL/SQL stack.
          </li>
          <li>
            Created a customer requested file upload web app with a tech stack in Vue.js, Node.js, Hapi.js, and Oracle PL/SQL
          </li>
          <li>
            Tools: Oracle EBS // Vue.js, Node.js, Groovy/Grails // Oracle PL/SQL, Linux, Java, SQL // Cloud Foundry
          </li>
        </ul>
      </div>
      <div
        className="card-footer"
      >
        <span>
          Jan 2017 - July 2018 | St. Louis, MO
        </span>
      </div>
    </div>
    <div
      className="card emotion-4"
    >
      <div
        className="card-content emotion-3"
      >
        <div
          className="row emotion-2"
        >
          <div
            className="col s12 m2 emotion-1"
          >
            <a
              href="https://wwt.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                className=" gatsby-image-wrapper"
                style={
                  Object {
                    "maxHeight": "100%",
                    "maxWidth": null,
                    "overflow": "hidden",
                    "position": "relative",
                  }
                }
              >
                <div
                  aria-hidden={true}
                  style={
                    Object {
                      "paddingBottom": "63.83896477354421%",
                      "width": "100%",
                    }
                  }
                />
                <img
                  alt=""
                  aria-hidden={true}
                  src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEj"
                  style={
                    Object {
                      "height": "100%",
                      "left": 0,
                      "objectFit": "contain",
                      "objectPosition": "center",
                      "opacity": 1,
                      "position": "absolute",
                      "top": 0,
                      "transitionDelay": "500ms",
                      "width": "100%",
                    }
                  }
                />
                <picture>
                  <source
                    sizes="(max-width: 800px) 100vw, 800px"
                    srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/wwt.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/wwt.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/wwt.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/wwt.jpg 1391w"
                  />
                  <img
                    alt="WWT Logo"
                    loading="lazy"
                    onLoad={[Function]}
                    sizes="(max-width: 800px) 100vw, 800px"
                    src="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg"
                    srcSet="/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/wwt.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/wwt.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/wwt.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/wwt.jpg 1391w"
                    style={
                      Object {
                        "height": "100%",
                        "left": 0,
                        "objectFit": "contain",
                        "objectPosition": "center",
                        "opacity": 0,
                        "position": "absolute",
                        "top": 0,
                        "transition": "opacity 500ms",
                        "width": "100%",
                      }
                    }
                  />
                </picture>
                <noscript
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<picture><source srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/wwt.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/wwt.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/wwt.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/wwt.jpg 1391w\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" srcset=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/fd013/wwt.jpg 200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/25252/wwt.jpg 400w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg 800w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/0ff54/wwt.jpg 1200w,
						  /static/a68ec0a3134b8610c476cd0fa3c0cf7e/6d185/wwt.jpg 1391w\\" src=\\"/static/a68ec0a3134b8610c476cd0fa3c0cf7e/2f1b1/wwt.jpg\\" alt=\\"WWT Logo\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                    }
                  }
                />
              </div>
            </a>
          </div>
          <div
            className="card-title"
          >
            <a
              className="teal-text hoverline"
              href="https://www.wwt.com/"
              rel="noopener noreferrer"
              target="_blank"
            >
              World Wide Technology
            </a>
          </div>
          <div
            className="role brown-text"
          >
            Developer
          </div>
        </div>
        <hr
          className="MuiDivider-root divider MuiDivider-middle"
        />
        <p
          className="subtitle"
        >
          <em
            className="text-description"
          >
            Technology and supply chain services provider
          </em>
        </p>
        <h6>
          Software Engineering
        </h6>
        <ul>
          <li>
            2+ years of programming experience as an Oracle PL/SQL Developer in Analysis, Design, and Implementation of Business Applications using the Oracle Relational Database Management System (RDBMS)
          </li>
          <li>
            Involved in all phases of the SDLC (Software Development Life Cycle) from analysis, design, development, testing, implementation and maintenance with timely delivery against deadlines
          </li>
          <li>
            Experience with data flow diagrams, data dictionary, entity relation modeling, and design techniques
          </li>
          <li>
            Heavily involved in yearlong Oracle EBS upgrade from 11i to 12.2.4
          </li>
          <li>
            Tools: Oracle EBS // Oracle PL/SQL, Linux, Java, SQL
          </li>
        </ul>
      </div>
      <div
        className="card-footer"
      >
        <span>
          March 2014 - Dec 2016 | St. Louis, MO
        </span>
      </div>
    </div>
  </div>
</section>
`;
